src/player_hardcoded_effects.cpp

```diff
@@ -1301,5 +1304,76 @@ void player::hardcoded_effects( effect &it )
             hp_cur[i] += 10;
         }
         mod_pain( -10 );
+    } else if( id == effect_toxin_buildup ) {
+        // Loosely based on toxic man-made compounds (mostly pesticides) which don't degrade
+        // easily, leading to build-up in muscle and fat tissue through bioaccumulation.
+        // Symptoms vary, and many are too long-term to be relevant in C:DDA (i.e. carcinogens),
+        // but lowered immune response and neurotoxicity (i.e. seizures, migraines) are common.
+
+        if( in_sleep_state() ) {
+            return;
+        }
+        // Modifier for symptom frequency.
+        // Each symptom is twice as frequent for each level of intensity above the one it first appears for.
+        int mod = 1;
+        switch( intense ) {
+            case 3:
+                // Tonic-clonic seizure (full body convulsive seizure)
+                if( one_turn_in( 3_days ) && !has_effect( effect_valium ) ) {
+                    add_msg_if_player( m_bad, _( "You lose control of your body as it begins to convulse!" ) );
+                    time_duration td = rng( 30_seconds, 4_minutes );
+                    add_effect( effect_motor_seizure, td );
+                    add_effect( effect_downed, td );
+                    add_effect( effect_stunned, td );
+                    if( one_in( 3 ) ) {
+                        add_msg_if_player( m_bad, _( "You lose conciousness!" ) );
+                        fall_asleep( td );
+                    }
+                }
+                mod *= 2;
+            /* fallthrough */
+            case 2:
+                // Myoclonic seizure (muscle spasm)
+                if( one_turn_in( 2_hours / mod ) && !has_effect( effect_valium ) ) {
+                    std::string limb = random_entry<std::vector<std::string>>( { "arm", "hand", "leg" } );
+                    add_msg_if_player( m_bad,
+                                       _( string_format( "Your %s suddenly jerks in an unexpected direction!", limb ) ) );
```

Comment:
`limb` in not translated.

Reply:
Thank you for your patience, I understand finally; the string has to be translated before formatting so that other languages can properly reorder the sentence structure.

Reply:
Well... not exactly, the string has to be translated before formatting because only the unformatted string gets extracted. The string extraction utility does not know what the formatted string would be, so it's impossible for it to extract the formatted string to be translated by the translators.

Reply:
Updated it, is it correct now?